enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationService.java

```diff
@@ -234,63 +234,6 @@ public List<DocumentVerificationEntity> submitDocuments(DocumentSubmitRequest re
         return docsVerifications;
     }
 
-    /**
-     * Starts the verification process
-     *
-     * @param ownerId Owner identification.
-     * @param identityVerification Identity verification.
-     * @throws RemoteCommunicationException In case of remote communication error.
-     * @throws DocumentVerificationException In case of business logic error.
-     */
-    @Transactional
-    public void startVerification(OwnerId ownerId, IdentityVerificationEntity identityVerification) throws DocumentVerificationException, RemoteCommunicationException {
-        List<DocumentVerificationEntity> docVerifications =
-                documentVerificationRepository.findAllDocumentVerifications(identityVerification,
-                        Collections.singletonList(DocumentStatus.VERIFICATION_PENDING));
-
-        List<DocumentVerificationEntity> selfiePhotoVerifications =
-                docVerifications.stream()
-                        .filter(entity -> DocumentType.SELFIE_PHOTO.equals(entity.getType()))
-                        .collect(Collectors.toList());
-
-        // If not enabled then remove selfie photos from the verification process
-        if (!identityVerificationConfig.isVerifySelfieWithDocumentsEnabled()) {
-            docVerifications.removeAll(selfiePhotoVerifications);
-        }
-
-        documentProcessingService.pairTwoSidedDocuments(docVerifications);
-
-        List<String> uploadIds = docVerifications.stream()
-                .map(DocumentVerificationEntity::getUploadId)
-                .collect(Collectors.toList());
-
-        final DocumentsVerificationResult result = documentVerificationProvider.verifyDocuments(ownerId, uploadIds);
-        final String verificationId = result.getVerificationId();
-        final DocumentVerificationStatus status = result.getStatus();
-        logger.info("Verified documents upload ID: {}, verification ID: {}, status: {}, {}", uploadIds, verificationId, status, ownerId);
-        auditService.auditDocumentVerificationProvider(identityVerification, "Documents verified: {} for user: {}", status, ownerId.getUserId());
-
-        moveToPhaseAndStatus(identityVerification, IdentityVerificationPhase.DOCUMENT_VERIFICATION, IdentityVerificationStatus.IN_PROGRESS, ownerId);
-
-        docVerifications.forEach(docVerification -> {
-            docVerification.setStatus(DocumentStatus.VERIFICATION_IN_PROGRESS);
```

Comment:
My main objection is this. `VERIFICATION_IN_PROGRESS` is always set, whatever the result is.

Reply:
Result of what? We are just starting verification of documents. If you remove this call, then method `checkVerificationResult` will never work.

Reply:
Ah, I see, this is the sync vs. async calling, right?

Reply:
My concern is what this does to the frontend. The frontend may wait for some time and be still stuck in old state during verification.

Perhaps we can set:

```
moveToPhaseAndStatus(identityVerification, IdentityVerificationPhase.DOCUMENT_VERIFICATION, IdentityVerificationStatus.IN_PROGRESS, ownerId);
```
and 

```          docVerification.setStatus(DocumentStatus.VERIFICATION_IN_PROGRESS);
```

in another transaction before starting the verification? This way the frontend will obtain correct state during verification. After your change there will be no way to find out for frontend that verification is in progress, right?

Reply:
> It's quite a big change, so we'll need to test this thoroughly.

Exactly. I do not push this change. I would like to postpone it, if possible, when we stabilize the current issues.

Reply:
`moveToPhaseAndStatus(identityVerification, IdentityVerificationPhase.DOCUMENT_VERIFICATION, IdentityVerificationStatus.IN_PROGRESS, ownerId);` was kept.

Reply:
`VerificationProcessingBatchService` kept to enable async in the future.